test/ci/template-inventory.j2

```diff
@@ -0,0 +1,29 @@
+[OSEv3:vars]
+ansible_become=true
```

Comment:
Why not put these in the group_vars file?

Reply:
Done